wg-serving/README.md

```diff
@@ -49,4 +49,42 @@ The WG Serving will operate in several workstreams with different areas of focus
 - [WS Autoscaling](https://docs.google.com/document/d/1QsN4ubjerEqo5L4bQamOFFS2lmCv5zNPis2Z8gcIITg/edit?usp=sharing)
 - WS MultiHost/MultiNode
 - [WS Orchestration](https://docs.google.com/document/d/1hbEx3ZEqdXCqWH9RL3uy9FIy35B8pFJ5KiK3HsOz2FE/edit?usp=sharing)
+
+
+## How to ...
+
+### subscribe to WG events
+
+1. Join the [`dev@kubernetes.io`](https://groups.google.com/a/kubernetes.io/g/dev) and [`wg-serving@kubernetes.io`](https://groups.google.com/a/kubernetes.io/g/wg-serving) for needed permissions.
+2. You should have meeting invites in your inbox after some delay. Make sure to fish them out of Junk/Spam and accept and invite so it will be added to the calendar
```

Comment:
(nit)
```suggestion
2. You should have meeting invites in your inbox after some delay. Make sure to fish them out of Junk/Spam and accept the invite so it will be added to your calendar.
```